src/auto-complete.jsx

```diff
@@ -64,7 +64,13 @@ const AutoComplete = React.createClass({
       open: this.props.open,
     };
   },
-
+  componentWillReceiveProps: function(nextProps) {
+    if (this.state.searchText !== nextProps.searchText) {
```

Comment:
It does work with .props though, I just tested with docs and with my code. I'll push it again